certs/rpk/include.am

```diff
@@ -0,0 +1,9 @@
+# vim:ft=automake
+# All paths should be given relative to the root
+#
+
+EXTRA_DIST += \
+         certs/rpk/client-cert-rpk.der \
+         certs/rpk/client-ecc-cert-rpk.der \
```

Comment:
Generation of these certs needs automated with certs/renewcerts.sh if possible. Eventually they will get out of date.

Reply:
Added code for updating those certs in certs/renewcerts.sh.